src/components/page/Header/Header.tsx

```diff
@@ -64,7 +67,15 @@ export class HeaderComponent extends React.Component<HeaderProps, State> {
       isOpen: false,
     };
   }
-
+  public componentWillReceiveProps(nextProps: any) {
```

Comment:
We should not do this here.  Instead, we need to supply a custom LogOut component to the `gatekeeper` package.  dditionally, let us try and not use the now deprecated `componentWillReceiveProps`, yes?

Reply:
At the very very least this should be `HeaderProps`, no?